example/initial.json

```diff
@@ -1,2451 +1,2530 @@
 {
-  "running": true,
-  "winner": "",
-  "council": {
-    "coffer": 0,
-    "council": [],
-    "senate": [],
-    "type": "council",
-    "can_bounty": true
-  },
-  "type": "state",
-  "day": 0,
-  "board": {
-    "unit_board": [
-      [
-        {
-          "attributes": {
```

Comment:
Please clarify: what is `them`? Are you referring to all fields of state, or just the board?

Reply:
All fields the only top level properties are attributes and class so to save on the nesting you should be able to put the fields on the top level.

Reply:
I think this would increase complexity since we cannot just assume all k/v pairs are attributes . We would likely need a soft convention to mark attributes (i.e., `_VAR` is an attribute and we cant have anything that starts with an _ that isnt an attribute) which I smells bad.

Reply:
I would assume anything other than the class field is an attribute no need to add a _VAR suffix.